src/utils/helperFunctions.ts

```diff
@@ -158,10 +180,11 @@ export const getGlobalChartData = (
                 compareAsc(data.date, endDate) === -1,
         );
 
-        const chartData = countryData.map((data) => {
+        const chartData = countryData.map((data, index) => {
```

Comment:
This is a very minor thing but just to make things consistent with the function above you can change `index` -> `idx`